views/partials/footer.ejs

```diff
@@ -27,14 +28,14 @@
       ></a>
     </div>
     <div class="page-links">
-      <a href="/">Shine Premium</a>
-      <a
-        href="https://images.ctfassets.net/awpxl2koull4/YaA04LXbkyUsyGggOS22I/f7734cca463599e1e359e16c3d7c21f2/Shine2019HustleAndHeartReport.pdf"
-        >Hustle & Heart Report</a
+      <a href="/">Join Shine</a>
+      <a href="https://premium.shinetext.com/quiz/start"
+        >Take the Self-Care Style Quiz</a
       >
       <a href="/advice">Get Advice</a>
       <a href="/referrals">Invite Friends</a> <a href="/squad">The Squad</a>
-      <a href="/jobs">Careers</a> <a href="/faq">FAQ</a>
+      <a href="https://careers.shinetext.com/">Careers</a>
```

Comment:
Any particular reason you're making this change from /jobs? The /jobs path should already be setup on production to redirect to this careers page. If it's not working on your local setup, there's probably an environment variable that you'd need to also set for yourself.

Reply:
ohh yea, I was clicking it from my local so it didn't redirect. should I change it back?